packages/salesforcedx-vscode-core/src/telemetry/MetricsReporter.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { TelemetryService } from '@salesforce/salesforcedx-utils-vscode';
+import * as vscode from 'vscode';
+import { BASE_EXTENSION, EXPANDED_EXTENSION, EXT_PACK_STATUS } from '../constants';
+
+export enum EXT_PACK_TYPES {
+  BASE = 'BASE',
+  EXPANDED = 'EXPANDED',
+  BOTH = 'BOTH',
+  NONE = 'NONE'
+};
+
+export class MetricsReporter {
+  private static telemetryService = TelemetryService.getInstance();
```

Comment:
any reason to have this as a property on the class?  You can just `TelemetryService.getInstance()` when needed.

Reply:
good point! I'll swap it